safe/impact_function/test/test_impact_function.py

```diff
@@ -216,7 +252,7 @@ def test_impact_function_behaviour(self):
         impact_function.exposure = exposure_layer
         impact_function.hazard = hazard_layer
         impact_function.prepare()
-        self.assertEqual(impact_function.name, 'Flood Polygon On Road Line')
+        self.assertEqual(impact_function.name, 'Flood Polygon On Roads Line')
         self.assertEqual(impact_function.title, 'be affected')
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->